include/zephyr/net/wifi_mgmt.h

```diff
@@ -38,7 +38,7 @@ extern "C" {
 #ifdef CONFIG_WIFI_MGMT_SCAN_SSID_FILT_MAX
 #define WIFI_MGMT_SCAN_SSID_FILT_MAX CONFIG_WIFI_MGMT_SCAN_SSID_FILT_MAX
 #else
-#define WIFI_MGMT_SCAN_SSID_FILT_MAX 0
+#define WIFI_MGMT_SCAN_SSID_FILT_MAX 1
```

Comment:
Did you see any issues with `0`? Or is this just from experience?

Reply:
Presumably the define was added because `CONFIG_WIFI_MGMT_SCAN_SSID_FILT_MAX` is not always defined.
`0` is flat out incorrect as an array length, see #11594